doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_custom_pcb.rst

```diff
@@ -22,7 +22,8 @@ We highly recommend using the PCB layouts and component values provided by Nordi
 * For optimal performance, the output impedance of the **P6** and **P7** ports should match the PCB and external device pin impedance.
 * Use one of the following power supply options:
 
-  * VDDH higher than 2.2V
+  * VDDH higher than 2.05V.
+    Between xx V and 2.05V with limitations on power consumption.
```

Comment:
@tjora Hey Sigve, I'd need some more info here

Reply:
Just remove the line "    Between xx V and 2.05V with limitations on power consumption.", we can add it in later when we know the details and would like to describe them.